borg-nix-shell.el

```diff
@@ -25,31 +25,32 @@
 
 ;;; Variables:
 
-(defvar borg-nix-shell-build-use-pure-shell
-  t
-  "Determine if nix-shells should be pure.")
+(require 'borg)
+
+(defcustom borg-nix-shell-build-use-pure-shell t
+  "Determine if nix-shells should be pure."
+  :type 'boolean
+  :group 'borg)
```

Comment:
I'm not sure this has any effect, since Borg runs the build in an inferior Emacs started with `--batch`, which implies `-Q`.  This is something that needs to be solved at Borg level, see https://github.com/emacscollective/borg/issues/78, especially https://github.com/emacscollective/borg/issues/78#issuecomment-433636542.    

(Turning a defvar into a defcustom is not the issue here, the problem is that the *variable* itself is useless)

Reply:
Yes, I've noticed that `make` with borg stuff doesn't always work well :-(